foreman/foreman.spec

```diff
@@ -158,6 +160,8 @@ BuildRequires: %{?scl_prefix}rubygem(deep_cloneable) >= 2.2.2
 BuildRequires: %{?scl_prefix}rubygem(deep_cloneable) < 3.0
 BuildRequires: %{?scl_prefix}rubygem(validates_lengths_from_database) >= 0.5
 BuildRequires: %{?scl_prefix}rubygem(validates_lengths_from_database) < 1.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.5
```

Comment:
I think foreman should no longer be foreman-tasks dependency, it's like with fog, rails or other foreman dependencies.